src/controllers/user.js

```diff
@@ -1,4 +1,4 @@
-import bcrypt from 'bcrypt';
+import bcrypt from 'bcryptjs';
```

Comment:
did you have any conversation with your team on why you switched to bcrypt js? Please do this on slack and agree if its the best way forward. Support your decision to do so

Reply:
okay